scripts/apply-migrations.sh

```diff
@@ -0,0 +1,3 @@
+sea-orm-cli migrate up
+
+#  sea-orm-cli generate entity -t <table_name>
```

Comment:
there's a few more flags you need. you also have to change a few column types (i8 -> bool, Vec<u8> -> Uuid, and some others) manually. check the README.